manim/_config/main_utils.py

```diff
@@ -150,14 +153,38 @@ def _parse_args_cfg_subcmd(args):
     return parsed
 
 
+def _parse_args_plugins(args):
+    """Parse arguments of the form 'manim plugins <args>'."""
+    parser = argparse.ArgumentParser(
+        description="Utility command for managing plugins",
+        prog="manim plugins",
+        epilog="Made with <3 by the manim community devs",
+        usage=("%(prog)s -h -u -l"),
+    )
+
+    parser.add_argument(
+        "-l",
+        "--list",
+        action="store_true",
+        help="Lists all available plugins",
+    )
+    parser.add_argument(
+        "-u", "--update", action="store_true", help="Updates plugins/__init__.py"
+    )
+    parsed = parser.parse_args(args[2:])
+    parsed.cmd = "plugins"
+    return parsed
+
+
 def _parse_args_no_subcmd(args):
     """Parse arguments of the form 'manim <args>', when no command is present."""
     parser = argparse.ArgumentParser(
         description="Animation engine for explanatory math videos",
         prog="manim",
         usage=(
             "%(prog)s file [flags] [scene [scene ...]]\n"
-            "       %(prog)s {cfg,init} [opts]"
+            "       %(prog)s {cfg,init} [opts]\n"
+            "       %(prog)s plugins [flags]"
```

Comment:
BTW, I am guilty of the fact that adding subcommands is a pain in the ass :sweat_smile: 

I am also guilty of the fact that this particular line is super messy and hard-coded.

May I suggest (for now):
```suggestion
            "       %(prog)s {cfg,init,plugins} [opts]\n"
```